src/config/config.py

```diff
@@ -0,0 +1,23 @@
+from collections import namedtuple
+
+from config.paths import DIR_NUSCENES
+
+Resolution = namedtuple('Resolution', ['height', 'width'])
+
+DATASET_ROOT = {'nuscenes': DIR_NUSCENES}
+TRAIN_RESOLUTION = {'nuscenes': Resolution(height=448, width=800)}
+
+
+def _get_dataset_param(dataset, dict):
```

Comment:
In general, avoid using built-in names for variables (`dict` in this case), as it can have unexpected consequences.

Reply:
fixed